code/modules/power/smes.dm

```diff
@@ -224,8 +224,10 @@
 
 	var/goal = (delta_power < 0) ? (charge) : (capacity - charge)
 	time = world.time + (delta_power ? ((goal / abs(delta_power)) * (world.time - last_time)) : 0)
-	// If it is negative - we are discharging
-	if(delta_power < 0)
+
+	if(input_cut == TRUE) // Cannot charge if input wire cut
```

Comment:
if this is a boolean then check it this way:
`if(input_cut)`

Reply:
oh duh right
```suggestion
	if(input_cut) // Cannot charge if input wire cut
```